src/platform/forms-system/src/js/utilities/data/findDuplicateIndexes.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Find duplicates in an array of objects and return an array of indexes for the
+ *  duplicates only
+ * @param {Object} formData - Data array (ArrayField data)
+ * @param {String} dataKey - key of data of interest to find duplicates
+ * @return {Array} - Array of indexes of the duplicates only, not the first
+ *  entry
+ * @example
+ * findDuplicateIndexes([{x:'one'},{x:'two'},{x:'one'},{x:'two'}], 'x')
+ * // => [2, 3]
+ */
+export default function(formData = [], dataKey = '') {
```

Comment:
```suggestion
export default function(formData = [], dataKey) {
```

Why default the `dataKey` to a blank string?

Reply:
I was trying to maintain the `type` of the parameter as a string, but leaving it `undefined` is good too since the next like checks if it's falsy.